tools/aa/commands/WorkspaceCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.google.startupos.tools.aa.commands;
+
+import com.google.startupos.common.FileUtils;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+import com.google.startupos.common.flags.Flags;
+import java.io.IOException;
+import java.nio.file.Paths;
+import javax.inject.Inject;
+
+public class WorkspaceCommand implements AaCommand {
+  @FlagDesc(
+    name = "force",
+    description = "Force workspace switching creating workspace if non-existent"
+  )
+  public static Flag<Boolean> force = Flag.create(false);
+
+  @FlagDesc(name = "ws", description = "Workspace name to switch to", required = true)
```

Comment:
We'd like to have:
`aa workspace <workspace name>`
`aa workspace -f <workspace name>`